.env-testnet-sample

```diff
@@ -50,15 +50,11 @@ LTC_RPC_PASS=sparkswapltc
 
 # External IP Addresses
 #
-# In order for your lightning nodes to be found by other users on the network
-# you must set an external address for each chain. If you are hosting all containers
-# on the same machine, then these will be the same.
+# This flag sets a static IP address for your LND instance. This address can either
```

Comment:
We should note in the comments that this is optional.

Reply:
@treygriffith I simply noted that the address is optional.